src/index.js

```diff
@@ -19,6 +21,14 @@ app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(validator());
 app.use(cors());
+app.use(passport.initialize());
+app.use(
+  session({
+    secret: 'keyboard cat',
```

Comment:
Should this be here? It looks like something that should be protected, no?

Reply:
I felt there was no need as it's not handling anything major. I would protect it